### PR TITLE
Update active_cname.js

### PR DESCRIPTION
**Pull request by OpAayush**
- [x] 申请该域名有实际的用途
- [x] 我已经阅读并遵守限定条款

---

I want to creat a site like uptimerobot.com

### CODE REVIEW

**Comment by OpAayush** [comment]
...

File: main/active_cname.js [hunk 1]
```diff
@@ -141,6 +135,13 @@ module.exports = {
   'zswangziye':'zswangziye.github.io',
   'duckwaline':'waline-3bg5otthj-deng-duck.vercel.app',
   'dengduck':'dengduck.netlify.app',
+  'spicydevs': 'spicydevs.pages.dev',
+  'cdn.spicydevs': 'spicydevs.github.io',
+  'sd': 'spicydevs.pages.dev',
+  'preview.spicydevs': 'spicydevs.github.io',
+  'minecraft': 'mcjs.pages.dev',
+  'servers': 'opaayush.github.io',
+  'uptime': '872b6773-e225-46b7-b6b9-25747d831c64.id.repl.co',
```
Comment (willin) [inline main/active_cname.js:144]: didn't change

**Review by willin — CHANGES_REQUESTED** [review]

File: main/active_cname.js (hunk 1 above)
Comment (OpAayush) [reply main/active_cname.js:144]: What to change?

**Comment by OpAayush** [comment]
I didn't understand what do you mean?
Do you have a discord account?

**Comment by OpAayush** [comment]
What's wrong in this?

**Comment by OpAayush** [comment]
If you don't want that then you can add text record for uptime.js.cool as 

> `replit-verification`

**Pull request closed without merging**